### PR TITLE
fix: use empty schema name by default

### PR DESCRIPTION
**Pull request by IlyaFaer**
The PR is based on another: https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/pull/24

### CODE REVIEW

File: google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py [hunk 1]
```diff
@@ -515,7 +527,7 @@ def get_table_names(self, connection, schema=None, **kw):
 FROM information_schema.tables
 WHERE table_schema = '{}'
 """.format(
-            schema
+            schema or ""
```
Comment (IlyaFaer) [inline google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py:530]: By default `schema` arg is `None`, so it tried to search for tables with schema `'None'`. Not sure how I've written this, it's not smart at all. Fixing it.

File: test/test_suite.py [hunk 2]
```diff
@@ -509,3 +549,181 @@ def _literal_round_trip(self, type_, input_, output, filter_=None):
...
+                ]
+            else:
+                tables = insp.get_table_names(schema)
+            table_names = [t for t in tables if t not in _ignore_tables]
+
+            if order_by == "foreign_key":
+                answer = ["users", "user_tmp", "email_addresses", "dingalings"]
+                eq_(table_names, answer)
+            else:
+                answer = ["dingalings", "email_addresses", "user_tmp", "users"]
```
Comment (IlyaFaer) [inline test/test_suite.py:728]: `user_tmp` is a temporary table, adding it into expected results

**Comment by IlyaFaer** [comment]
@larkee, this PR fixes two more reflection tests:

Безымянный

**Review by larkee — COMMENTED** [review]
Changes introduced in this PR (i.e. schema name and table name test override) LGTM :+1: 

Approval pending the base PR

**Comment by AVaksman** [comment]
Merging to unblock further work

**Review by AVaksman — APPROVED** [review]

**Pull request merged**